SPECS-EXTENDED/rdma-core/rdma-core.spec

```diff
@@ -0,0 +1,747 @@
+Summary:        RDMA core userspace libraries and daemons
+Name:           rdma-core
```

Comment:
What happens to the rdma-core package out of SPECS?

Reply:
I wasn't part of the original discussion, but from what I understand, the rdma-core package in SPECS is at version 49.1, while this version is 55.0 (MOFED requires the latest version). As a general rule, we avoid major version upgrades in Azure Linux updates to prevent potential disruptions to consumers' applications. Therefore, I assume the rdma-core package in SPECS will remain unchanged for AZL 3.0 or, at most, might be upgraded to version 49.x.

Reply:
Anything in specs extended will not be actively maintained for CVE's/etc.

Seems to me we have two options here:  Upgrade the main one now, or let this version simmer in Extended for awhile and then upgrade in core.

I'd be more inclined to rip the band-aid and upgrade the 3.0 SPECS version first. There are few customers using 3.0.  I'm not sure how many are using rdma-core.  Has anyone evaluated what happens if we upgrade the core version (and yeah...it's a big jump).  In particular have we looked at any compatibility concerns?  I think we should discuss the options here.

Reply:
`rdma-core` is one of the MOFED dependencies and each of its major releases introduced new mellanox support and bug fixes which are critical to MOFED.

It seems the original plan was to have the latest version in extended for a while and then promote it to core. If we go through upgrading it in core, do you have any idea how we can validate if version increase affects any customers?